pdpipe/core.py

```diff
@@ -652,6 +655,17 @@ def __init__(self, stages, transformer_getter=None, **kwargs):
     def __getitem__(self, index):
         if isinstance(index, slice):
             return PdPipeline(self._stages[index])
+
+        if isinstance(index, list):
```

Comment:
do you want to check for a list, or a list of strings?

Reply:
I was not sure if it's right to force the `label` param to be an `str`.